### ISSUE TITLE
Activities are not in sync in multisite

### ISSUE THREAD

**Issue by SID177** [rtCamp/rtMedia#1591]
**Note:** In multisite setup, BuddyPress by default shows activity feed only on main site. To have subsite specific activity feed, add `define( 'BP_ENABLE_MULTIBLOG', true );` in `wp-config.php`.
Ref - https://codex.buddypress.org/getting-started/customizing/bp_enable_multiblog/

When rtMedia is active and above code is added, activity (with media) added from main site won't be displayed on a subsite.
This is not the case when rtMedia is not active.

The fix added for `Broken media URL with multisite` issue in rtMedia v4.5.3 caused this to happen.

**`Broken media URL with multisite`:** Activity added from main site was showing on a subsite with broken media links. So as a fix, activity of a particular site will be displayed only on that site.

The point here is when rtMedia is not active, the activity is displayed on subsite without broken media link. So we need to check if some code is altering rtMedia's media links.

**Comment by SID177** [comment]
@kjnanda In a multisite setup (sub directory), I'm not able to upload media from a subsite. Can you please confirm if you're also getting similar issue?

**Comment by SID177** [comment]
In the latest rtMedia v4.6.2, the activity is in sync with subsite. But when rtMedia is active it gives broken media link, which needs to be fixed.

**Comment by pooja-muchandikar** [comment]
@SID177 

Cross-checked this issue and its working as expected.

```
Test Environment
rtMedia: v4.6.3 pre-release branch
BuddyPress: v6.0.0
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
